forms/src/app/login-form/login-form.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.css']
+})
+export class LoginFormComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onSubmit(value) {
+    console.log(JSON.stringify(value));
```

Comment:
Might be better to display this value in the app itself, rather than a console log statement. Maybe within in the login-form-component template there could be a simple section for displaying the users input on submission. Console.log kinda gets buried, especially if they have Augury tab open in dev tools and not the Console tab.